A warning...

```
In file included from .pio\libdeps\nodemcuv2\aWOT\src\aWOT.cpp:23:0:
.pio\libdeps\nodemcuv2\aWOT\src\aWOT.cpp: In member function 'void Response::set(const char*, const char*)':
.pio\libdeps\nodemcuv2\aWOT\src\aWOT.h:72:52: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
 #define SIZE(array) (sizeof(array) / sizeof(*array))
                                                    ^
.pio\libdeps\nodemcuv2\aWOT\src\aWOT.cpp:104:25: note: in expansion of macro 'SIZE'
   if (m_headersCount >= SIZE(m_headers)) {
```


I hope it can be fixed, thanks!

--- a/library.properties
+++ b/library.properties
@@ -1,5 +1,5 @@
 name=aWOT
-version=3.2.2
+version=3.2.3
 author=Lasse Lukkari <lasse.lukkari@gmail.com>
 maintainer=Lasse Lukkari <lasse.lukkari@gmail.com>
 sentence=Arduino web server library.

--- a/src/aWOT.cpp
+++ b/src/aWOT.cpp
@@ -114,7 +114,7 @@ void Response::sendStatus(int code) {
 }
 
 void Response::set(const char *name, const char *value) {
-  if (m_headersCount >= SIZE(m_headers)) {
+  if (m_headersCount >= SERVER_MAX_HEADERS) {
     return;
   }
 
@@ -827,8 +827,8 @@ void Request::push(uint8_t ch) {
   m_pushback[m_pushbackDepth++] = ch;
 
   // can't raise error here, so just replace last char over and over
-  if (m_pushbackDepth == SIZE(m_pushback)) {
-    m_pushbackDepth = SIZE(m_pushback) - 1;
+  if (m_pushbackDepth == SERVER_PUSHBACK_BUFFER_SIZE) {
+    m_pushbackDepth = SERVER_PUSHBACK_BUFFER_SIZE - 1;
   }
 }
 

--- a/src/aWOT.h
+++ b/src/aWOT.h
@@ -69,7 +69,6 @@
 #endif
 
 #define P(name) static const unsigned char name[] PROGMEM
-#define SIZE(array) (sizeof(array) / sizeof(*array))
 
 class StreamClient : public Client {
  private:

